packager/app/packager_main.cc

```diff
@@ -325,9 +325,21 @@ base::Optional<PackagingParams> GetPackagingParams() {
   ChunkingParams& chunking_params = packaging_params.chunking_params;
   chunking_params.segment_duration_in_seconds = FLAGS_segment_duration;
   chunking_params.subsegment_duration_in_seconds = FLAGS_fragment_duration;
+  chunking_params.is_low_latency_dash = FLAGS_is_low_latency_dash;
   chunking_params.segment_sap_aligned = FLAGS_segment_sap_aligned;
   chunking_params.subsegment_sap_aligned = FLAGS_fragment_sap_aligned;
 
+  if (chunking_params.is_low_latency_dash && chunking_params.subsegment_duration_in_seconds > 0) {
+    // Low latency streaming requires data to be shipped as chunks,
+    // the smallest unit of video. Therefore, in low latency mode,
+    // each fragment will only contain one chunk, defaulting the
+    // fragment duration to the shortest time possible
+    // and making a user specified --fragment_duration irrelevant.
```

Comment:
Fixed the typo

Reply:
Can a chunk be a frame?

Reply:
Can a chunk be a frame?

Reply:
Yes, a chunk can contain a single frame up to a handful of frames. In this case, each chunk contains one frame.